.travis.yml

```diff
@@ -36,6 +36,8 @@ before_install:
     - echo $TRAVIS_BUILD_ID > __travis_build_id__.txt
     - nvm install stable
     - nvm use stable
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
```

Comment:
can you add a comment here for future readers? this is something specific for GUI testing in TravisCI but people is not usually aware of that...